docs/parameters.yaml

```diff
@@ -46,7 +46,10 @@ components: ["origin", "nsregistry", "director"]
 ---
 name: IssuerKey
 description: >-
-  The name of the file containing a service's private `issuer.jwk` key.
+  A filepath to the file containing a ecdsa-encrypted private key which later will be parsed
```

Comment:
I think it's more accurate to say "... containing a PEM-encoded ecdsa private key" -- is the key itself actually encrypted with ecdsa?

Reply:
I think yours is more accurate.